app/models/alchemy/page/page_scopes.rb

```diff
@@ -100,5 +96,35 @@ module Page::PageScopes
       #
       scope :sitemap, -> { from_current_site.published.contentpages.where(sitemap: true) }
     end
+
+    module ClassMethods
+
+      # All public pages
+      #
+      def published
+        publication_scope([
+          "#{table_name}.public_on <= ? AND".freeze,
```

Comment:
Style/IndentArray: Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.